helpers.php

```diff
@@ -17,6 +17,13 @@ function retry($retries, $fn)
     }
 }
 
+if (! function_exists('dayBefore')) {
+    function dayBefore($dateString)
+    {
+        return with(new \Carbon\Carbon($dateString))->subDay()->format("Y-m-d");
```

Comment:
Don't think you need `with()` in modern PHP. You can just
```php
return (new Carbon\Carbon($dateString))->subDay();
```

Also no need to format it. Always wiser to just return Carbon objects when you can.